tools/validators/instance_validator/validate/point.py

```diff
@@ -25,4 +25,12 @@ class Point(object):
   def __init__(self, point_name, value):
     super().__init__()
     self.point_name = point_name
-    self.present_value = value
+    self.present_value = self._ToString(value)
+
+  def _ToString(self, value):
+    if isinstance(value, bool):
+      return '{}'.format(value).lower()
```

Comment:
In this case, str(value) might be better, since there's no additional string content - you're just converting the value to a string.

Reply:
done